tools/sanity_checks.py

```diff
@@ -140,7 +140,7 @@
         'process-zconf.py',
     ],
 }
-NO_TABS_FILES = ['meson.build', 'meson_options.txt']
+NO_TABS_FILES = ['meson.build', 'meson_options.txt', 'meson.options']
```

Comment:
I'm slightly skeptical about this just because using this will immediately imply a minimum meson version of at least 1.1.0, which is a bit restrictive. It's not even available in Debian stable, let alone debian oldstable.

It's a very easy limitation to avoid, and avoiding it makes individual wraps more useful to a wider range of user.

On the other hand, it's possible a wrap legitimately needs newer versions -- various rust improvements, fs.relative_to(), dependency('atomic') since 1.7.0 (!!) -- so I don't know. I guess it's fine to allow, even if in most cases we'd still refrain from using it by default...

Reply:
This just adds the new options file (of which there a 6 in the repo atm (reflex, cpp-semver, mpark-patterns, pegtl, glpk, godot-cpp)) to the files, that are checked for tabs. This list just lists file names, that when encountered are checked.